book/05-distributed-git/sections/contributing.asc

```diff
@@ -412,7 +412,7 @@ To jessica@githost:simplegit.git
 
 This is called a _refspec_.
 See <<_refspec>> for a more detailed discussion of Git refspecs and different things you can do with them.
-Also notice the `-u` flag; this is short for `--set-upstream`, which configures the branches for easier pushing and pulling later.
+Also notice the `-u` flag; this is short for `--set-upstream-to`, which configures the branches for easier pushing and pulling later.
```

Comment:
Where does this change come from? The flag is still shown as `--set-upstream` in [the docs](https://git-scm.com/docs/git-push) and when I just ran `git push --help` with 2.10.2.